apps/studio/src/server/modules/resource/resource.service.ts

```diff
@@ -39,6 +44,34 @@ export const getPageById = (id: string) => {
     .executeTakeFirstOrThrow()
 }
 
+export const updatePageById = (
+  page: Partial<Omit<Page, 'id' | 'blobId'>> & { id: string },
+) => {
+  const { id, ...rest } = page
+  return db.transaction().execute((tx) => {
+    return tx
+      .updateTable('Resource')
```

Comment:
why are we not using constants for the table names?

Reply:
not doing this because kysely infers typings automatically (see ss below), so in this case, the table is already given as a suggestion. additionally, giving the wrong type (`resource` instead of `Resource`) will be an error.

the value of giving companion objects is 2 fold
- when the argument type is `string` but we want to say "only this subset of strings: `A | B | C`" and avoid errrors
- when we want to also infer a type 

in our case here, neither of this is true because kysely provides typing + auto-completion so i didn't see a big reason to do it + take on maint debt 

![image](https://github.com/opengovsg/isomer/assets/44049504/dc32b6c1-135d-444b-b8d3-127bbeb07a6e)